daemon/algod/api/client/restClient.go

```diff
@@ -402,8 +407,12 @@ func (client RestClient) AccountInformation(address string) (response v1.Account
 }
 
 // AccountInformationV2 gets the AccountData associated with the passed address
-func (client RestClient) AccountInformationV2(address string) (response generatedV2.Account, err error) {
-	err = client.get(&response, fmt.Sprintf("/v2/accounts/%s", address), nil)
+func (client RestClient) AccountInformationV2(address string, includeAppAssets bool) (response generatedV2.Account, err error) {
+	if includeAppAssets {
```

Comment:
```suggestion
func (client RestClient) AccountInformationV2(address string, includeCreatables bool) (response generatedV2.Account, err error) {
	if includeCreatables {
```
tiny nit, wdyt

Reply:
I agree with @cce here, and maybe consider writing this as:
```golang
func (client RestClient) AccountInformationV2(address string, includeCreatables bool) (response generatedV2.Account, err error) {
        var infoParams accountInformationParams
 	if includeCreatables {
              infoParams = accountInformationParams{Include: "all"}
        } else {
              infoParams = accountInformationParams{Include: "none"}
        }
        err = client.get(&response, fmt.Sprintf("/v2/accounts/%s", address), infoParams)
        return
}
```